Docs 3717 fix titles inconsistencies

BcnCarlos opened the pull request:
Fix 33 Interactive quickstart titles for consistency across all of them 

From 

Javascript quickstart: "Log In to the Application"

Angular quickstart: "Add Login to Your Application"

Javascript interactive quickstart: "Add login to your application"

To a standard of 

Add Login to your application

<!---
Pull Requests for Quickstart Guides can still be submitted here, but most other documentation content is no longer hosted on GitHub and therefore no longer open-sourced. If you are an Auth0 employee trying to make a change, please [submit a ticket](https://auth0team.atlassian.net/servicedesk/customer/portal/9). Thank you!
--->

BcnCarlos committed:
Update 01-login.md

Fix titles inconsistencies

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

BcnCarlos committed:
Update interactive.md

Fix title inconsistency 

Add Login to a Laravel Application 

to

Add Login to your Laravel application

BcnCarlos committed:
Update interactive.md

Fix inconsistency 

From

Add login to your Python Flask app

to

Add Login to your Python Flask application

BcnCarlos committed:
Update interactive.md

Fix title inconsistencies

Add login to your Ruby on Rails app

to

Add Login to your Ruby on Rails application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add Login to Your Angular App

To
Add Login to your Angular application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add login to your Flutter app

To
Add Login to your Flutter application

BcnCarlos committed:
Update interactive.md

Fix inconsistency

From
Add Login to Your JavaScript App

To
Add Login to your JavaScript application

BcnCarlos committed:
Update interactive.md

Fix inconsistency

From

Add login to your Vue app

To

Add Login to your Vue application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add Login to your React App

To
Add Login to your React application

BcnCarlos committed:
Update interactive.md

Fix inconsistency 

From 
Add Login to your Express App

To
Add Login to your Express application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add Login to .NET Android and iOS App

To
Add Login to your .NET Android and iOS application

BcnCarlos committed:
Update interactive.md

Fix inconstencies

From
Add login to your Android app

To
Add Login to your Android application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add Login to your React Native App

To
Add Login to your React Native application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add login to your Flutter app

To
Add Login to your Flutter application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add login to your Ionic Angular with Capacitor app

To
Add Login to your Ionic Angular with Capacitor application

BcnCarlos committed:
Update interactive.md

Fix title inconsistency

From
Add Login to your iOS or macOS app

To
Add Login to your iOS or macOS application

BcnCarlos committed:
Update interactive.md

updated title 

From 
Add Login to a MAUI application

To
Add Login to your MAUI application

BcnCarlos committed:
Update interactive.md

Fix titles

From
Add Login to your React Native App

To
Add Login to your React Native application

BcnCarlos committed:
Update interactive.md

Fix inconsistency

From
Add Login to your React Native App

To
Add Login to your React Native application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add Authorization to an ASP.NET Core Web API Application

To
Add Authorization to an ASP.NET Core Web API application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add authorization to an ASP.NET Owin Web API application.

To
Add Authorization to an ASP.NET Owin Web API application

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

Fix titles

From 
Add Authorization to a Django API Application

To
Add Authorization to a Django API application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add authorization to a Go application.
To
Add Authorization to a Go application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add Authorization to a Laravel Application
To
Add Authorization to a Laravel application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add Authorization to a Laravel Application
To
Add Authorization to a Laravel application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add endpoint authorization to your PHP application
To
Add Authorization to a PHP application

BcnCarlos committed:
Update interactive.md

Fix title

From
Add authorization to a Ruby on Rails API
To
Add Authorization to a Ruby on Rails API

BcnCarlos committed:
Update interactive.md

Fix title

From
Add authorization to a Spring Boot application
To
Add Authorization to a Spring Boot application

avanscoy commented on articles/quickstart/backend/aspnet-core-webapi/interactive.md:
"Application" should also be capitalized here as it is in the title. The rest of the words, except for articles and prepositions, are similar to our titles in docs.

avanscoy commented on articles/quickstart/native/android/interactive.md:
```diff
@@ -85,7 +85,7 @@ Run **Sync Project with Gradle Files** inside Android Studio or execute `./gradl
 
-## Add login to your app {{{ data-action=code data-code="MainActivity.kt#6:38" }}}
+## Add Login to your application {{{ data-action=code data-code="MainActivity.kt#6:38" }}}
 
```
^ This is a secondary heading, which is why it has ## so it does not need to be capitalized. "Add login to your application."

avanscoy commented on articles/quickstart/native/android/interactive.md:
```diff
@@ -117,7 +117,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
-## Add logout to your app {{{ data-action=code data-code="MainActivity.kt#40:52" }}}
+## Add Logout to your application {{{ data-action=code data-code="MainActivity.kt#40:52" }}}
 
```
^ This is a secondary heading, which is why it has ## so it does not need to be capitalized. "Add logout to an application."
I put "an" here instead of "your" but if all the secondary headings have "your", it should be fine.

avanscoy commented on articles/quickstart/backend/webapi-owin/interactive.md:
```diff
@@ -18,7 +18,7 @@ files:
 
-# Add authorization to an ASP.NET Owin Web API application.
+# Add Authorization to an ASP.NET Owin Web API application
 Auth0 allows you to add authorization to any kind of application. This guide demonstrates how to integrate Auth0 with any new or existing ASP.NET Owin Web API application using the `Microsoft.Owin.Security.Jwt` package.
```
Application

avanscoy commented on articles/quickstart/backend/aspnet-core-webapi/interactive.md:
```diff
@@ -20,7 +20,7 @@ files:
 ---
 
-# Add Authorization to an ASP.NET Core Web API Application
+# Add Authorization to an ASP.NET Core Web API application
```
Application

avanscoy commented on articles/quickstart/native/android/interactive.md:
```diff
@@ -11,7 +11,7 @@ github:
 ---
 
-# Add login to your Android app
+# Add Login to your Android application
```
Add Login to an Android Application

avanscoy commented on articles/quickstart/native/flutter/interactive.md:
```diff
@@ -22,7 +22,7 @@ github:
 ---
 
-# Add login to your Flutter app
+# Add Login to your Flutter application
```
Add Login to a Flutter Application

avanscoy commented on articles/quickstart/native/ionic-angular/interactive.md:
```diff
@@ -22,7 +22,7 @@ files:
 ---
 
-# Add login to your Ionic Angular with Capacitor app
+# Add Login to your Ionic Angular with Capacitor application
```
Add Login to an Ionic Angular with Capacitor Application

avanscoy commented on articles/quickstart/native/ios-swift/interactive.md:
```diff
@@ -21,7 +21,7 @@ contentType: tutorial
 
-# Add Login to your iOS or macOS app
+# Add Login to your iOS or macOS application
 
```
Add Login to an iOS or macOS Application

avanscoy commented on articles/quickstart/native/maui/interactive.md:
```diff
@@ -20,7 +20,7 @@ files:
 
-# Add Login to a MAUI application
+# Add Login to your MAUI application
 
```
You started removing "your" from the files above. Application sould be cap

avanscoy commented on articles/quickstart/native/maui/interactive.md:
```diff
@@ -135,7 +135,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
-## Add Login to Your Application {{{ data-action="code" data-code="MainPage.xaml.cs#25" }}}
+## Add Login to your application {{{ data-action="code" data-code="MainPage.xaml.cs#25" }}}
 
```
Secondary heading ^

avanscoy commented on articles/quickstart/native/maui/interactive.md:
```diff
@@ -167,7 +167,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
-## Add Logout to Your Application {{{ data-action="code" data-code="MainPage.xaml.cs#32" }}}
+## Add Logout to your application {{{ data-action="code" data-code="MainPage.xaml.cs#32" }}}
 
```
Secondary heading

avanscoy commented on articles/quickstart/native/net-android-ios/interactive.md:
```diff
@@ -21,7 +21,7 @@ files:
 
-# Add Login to .NET Android and iOS App
+# Add Login to your .NET Android and iOS application
 
```
Add Login to a .NET Android and iOS Application

avanscoy commented on articles/quickstart/native/net-android-ios/interactive.md:
```diff
@@ -130,7 +130,7 @@ This is an example of the XML representation of your `info.plist` file after you
 
-## Add Login to Your Application 
+## Add Login to your application 
 
```
Secondary heading

avanscoy commented on articles/quickstart/native/net-android-ios/interactive.md:
```diff
@@ -162,7 +162,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
-## Add Logout to Your Application
+## Add Logout to your application
 
```
Secondary heading

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update 01-login.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Update interactive.md

BcnCarlos committed:
Merge branch 'master' into DOCS-3717-fix-titles-inconsistencies

BcnCarlos committed:
Update interactive.md

Change 

From
Add Login to your Blazor Server application

To
Add Login to Your Blazor Server Application

BcnCarlos committed:
Update interactive.md

Change 

From
Add Login to your ASP.NET MVC application
To
Add Login to Your ASP.NET MVC Application

BcnCarlos committed:
Update interactive.md

Change 

From
Add Login to your Next.js application

To
Add Login to Your Next.js Application

BcnCarlos committed:
Update interactive.md

Change 

From 
Add Login to your PHP application
To
Add Login to Your PHP Application

BcnCarlos committed:
Update interactive.md

Change 

from
Add Authorization to an ASP.NET Core Web API Application

To
Add Authorization to Your ASP.NET Core Web API Application

BcnCarlos committed:
Update interactive.md

Change 

From
Add Authorization to a Flask API application

To
Add Authorization to Your Flask API Application
